### ISSUE TITLE
'Skip' transition needs alignment with general card transition animation

### ISSUE THREAD

**Issue by mbransn** [common-voice/common-voice#1042]
As spoken about today, preferred transition for sentence skip is to align with full sentence card transition animation (rather than just clip text refreshing).

**Comment by Gregoor** [comment]
I definitely see its value, there's just some implementation/visual difficulty:
Currently one sees the next up clip on the left with its sentence. Now if we reuse that transition for skipping we'd have to swap out that card, which might be noticed. Though since there's motion, it might only require a little stagecraft to make it seemless.
Another thought I had on this is that to me it makes visual sense that the cards move to the right and kind of "become" the pills (not sure if this was intended). Skipped sentences moving into the same direction would imo kill that narrative, wouldn't it?
So here's another idea (which is also easier to implement, just so that you know my bias): Move skipped cards away vertically and have another card sit underneath it. Interested to hear your thoughts on this 🙂

**Comment by m-branson** [comment]
@Gregoor I appreciate your deep narrative thoughts here. 

My recommendation: A card moves from left-to-right regardless of whether it is completed or not. If it is completed, it becomes a pill. If it is not completed, it does not become a pill. Therefore we should be giving skipped sentences the same animation as a card. e.g.; card=sentence rather than a card=clip ...or, every sentence is a card, they are not independent.

**Comment by jiya-singhal** [comment]
hey @MichaelKohler can you please assign this issue to me

**Comment by MichaelKohler** [comment]
Given that this issue is 6 years old, I wonder if this is still the case. @jiya-singhal wanna check if this still needs to be done first?

**Comment by ftyers** [comment]
The issue is that when you skip, the card just updates the text and doesn't have the animation. When you record there is an animation.

**Comment by ftyers** [comment]
Hi @VedantMadane can you send a screen recording of your PR working? Thanks!
